### ISSUE TITLE
Tested online receipt scanner, not working?

### ISSUE THREAD

**Issue by e-t-l** [aspose-ocr-cloud/aspose-ocr-cloud-android#2]
I wanted to test out Aspose OCR Cloud's ability before recommending it to a client, so I tried uploading a random receipt to the [Receipt Scanner web app](https://api.products.aspose.app/ocr/en/scan-receipt). Unless I'm doing something wrong, it isn't working. It uploads the image, says `Processing...` (with a button beneath it that says "Add to clipboard"; clicking it helpfully adds the string "Processing..." to my clipboard). After a few seconds, the Processing text disappears, and a green `Download` button is displayed (I would have expected to see the text output of the OCR in the web browser). However, when I do download it, I get a file named "Result" which contains nothing but a random alphanumeric string, presumably the hash value of something. For reference, I am attaching both the original receipt image I tried uploading, as well as the Result file. I have tried this multiple times from multiple browsers and devices. The outcome is always the same (note: the hash string in the Result file does change each time, so I'm guessing it's connected to the unique instance of upload/processing.)

TLDR: I upload an image of a receipt, it appears to be processed, but the output is gibberish and not resemblant to any text that could have been OCR'ed from the image.

[result.pdf](https://github.com/aspose-ocr-cloud/aspose-ocr-cloud-android/files/12489489/result.pdf) 
(If you don't want to click the PDF, it contains nothing but the string `fe38aff6-6eb2-4eaf-8b01-10c53b0373c7`.

TestReceipt image:
<image src="https://github.com/aspose-ocr-cloud/aspose-ocr-cloud-android/assets/40775958/21b9a3a8-35df-4c04-afd2-a1c343054a89" width=50%>

**Comment by Alexgolshtein** [comment]
Greetings!  Thanks for your response about the bug.  We will fix it as soon as possible and notify you immediately.

**Comment by AleksandrSudakov** [comment]
Hello! We have fixed this issue and you can check again.

**Comment by AleksandrSudakov** [comment]
Closed due to fix.

**Comment by e-t-l** [comment]
> Hello! We have fixed this issue and you can check again.

Thank you, it does indeed produce results now. Unfortunately, the quality of the results is disappointing. Using the original receipt image shared above, this is the result:
```
AMOUNT DUE

TOTAL

$ 18°25

6 ITEMS

nm JIta flaw

ah

=
SUBTOTAL

12

B-Taxable 61.0008

019
3.49 FB

craft Soda

09 FB

sorklng Flyrd tr

49 FB

Coconut Hater

ork Watermelon 4p

29 FB

ab Milk

8 FB

2 MITK

8 FB

VISA

yh

:

3
SUBTOTAL

13.12
```

Given that it didn't OCR the name of the vendor (Aldi) nor the receipt total ($13.25), it's not particularly useful as a receipt scanner app in its current form.

**Comment by Alexgolshtein** [comment]
Thank you for your review. At the moment, we are not able to extract all information on all checks, but this is fixable.

Could you please specify which fields (data) you would like to see in the result? Please list them in order of importance. (vendor, total).

In the next month, we will change our algorithm to make the data search and output results more accurate and take into account your requests.

P.S. Your receipts will also be useful to us as examples for testing, and we would be very glad if you send them to us.
